internal/build/cmd/tools/commands/spec/command_test.go

```diff
@@ -46,7 +46,7 @@ func TestBuild_zipfileUrl(t *testing.T) {
 			  "build_id": "1.0.0-ab7cd914",
 			  "projects": {
 				"opensearch": {
-				  "branch": "master",
+				  "branch": "main",
```

Comment:
@Xtansia Do we know if the branch change has happened? I was unable to find the artifact listed here.

Reply:
@harshavamsi This is just test data, not actually used other than to validate some logic that pulls out the archive url.